src/main/java/ro/developmentfactory/thedrones/entity/Drone.java

```diff
@@ -0,0 +1,37 @@
+package ro.developmentfactory.thedrones.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+public class Drone {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id_drone")
+    private Long idDrone;
```

Comment:
please use [UUIDs for the ids](https://www.baeldung.com/java-hibernate-uuid-primary-key). this way it will:
* be randomized, so that if exposed, the others cannot guess it
* and not be limited by the sequence max size - I know it might sound like this will never happen in production but it happens, I've seen it :)

Reply:
done